### PR TITLE
Add support for passing `-Dtests.` options to the gradle test process

### PR DESCRIPTION
**Pull request by mfussenegger**
In order to support setting seed, iterations and suite timeout via cli
argument.

### CODE REVIEW

**Comment by mfussenegger** [comment]
@seut please review

File: DEVELOP.rst [hunk 1]
```diff
@@ -110,25 +110,27 @@ And start Crate::
...
 
-    To enable logging for tests::
+    ./gradlew itest
 
-    ./gradlew -PtestLogging test
+    ./gradlew -PtestLogging :sql:test
 
-    Use @TestLogging(["<packageName1>:<logLevel1>", ...]) on your
-    test class or test method to enable more detailed logging.
+    ./gradlew test -Dtest.single='YourTestClass'
```
Comment (seut) [inline DEVELOP.rst]: pls also mention --tests=*Class.method for single test method execution

File: DEVELOP.rst [hunk 2]
```diff
@@ -110,25 +110,27 @@ And start Crate::
     ./app/build/install/crate/bin/crate
 
 
-Other common tasks are:
+Other common tasks are::
 
- - Running tests during development::
 
-    ./gradlew --parallel :sql:test -PtestForks=2 itest gtest
+    ./gradlew --parallel -PtestForks=2 :sql:test
```
Comment (seut) [inline DEVELOP.rst]: isn't `--parallel` and `-PtestForks`  somehow interfering ?

Comment (mfussenegger) [inline DEVELOP.rst]: never had a problem with that. Afaik `--parallel` is for the gradle build process and `-PtestForks` is for the actual test execution

**Pull request merged**